client/lib/analytics/tracker-buckets.ts

```diff
@@ -56,8 +57,8 @@ export const AdTrackersBuckets: { [ key in AdTracker ]: Bucket | null } = {
 	twitter: Bucket.ADVERTISING,
 	facebook: Bucket.ADVERTISING,
 
-	// Advertising trackers (only Jetpack Cloud):
-	linkedin: isJetpackCloud() ? Bucket.ADVERTISING : null,
+	// Advertising trackers (only Jetpack Cloud or on Jetpack Checkout):
+	linkedin: isJetpackCloud() || isJetpackCheckout() ? Bucket.ADVERTISING : null,
```

Comment:
Same comment as before but here. I don't know if there is a good place for this but it'd be even more reusable if we stored that array somewhere else and imported them into this file and the last. Could be overkill though